### PR TITLE
Fixed issue #58, extension install path needs to specify version in path

### PR DESCRIPTION
**Pull request by rkeithhill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # PowerShell Language Support for Visual Studio Code
 
 This extension provides rich PowerShell language support for Visual Studio Code.
-Now you write and debug PowerShell scripts using the excellent IDE-like interface
+Now you can write and debug PowerShell scripts using the excellent IDE-like interface
```
Comment (daviwil) [inline README.md]: hah, whoops!

**Comment by daviwil** [comment]
Thanks for fixing this!  Do you mind also updating examples\README.md since it also contains a reference to the path?

**Comment by rkeithhill** [comment]
Done.  Hope you don't mind me substituting `$Home` for `$env:USERPROFILE`.  `$Home` is shorter and should just as well when run from PowerShell.  And for that matter, `$env:USERPROFILE` only works in PowerShell as well.  

FYI, someone may be tempted to use `~` as an even shorter form of `$home` but `~` doesn't work when you are cd'd into a psdrive other than the filesystem.

**Comment by daviwil** [comment]
Nope, that's a great change, didn't know `$Home` was available!  Thanks for letting me know about the tilde context, another little PowerShell catch I didn't know about :)

**Pull request merged**